nekoyume/Assets/_Scripts/UI/Widget/Popup/BattleResultPopup.cs

```diff
@@ -111,6 +112,9 @@ public struct DefeatTextArea
         [SerializeField]
         private Button closeButton = null;
 
+        [SerializeField]
+        private Button stageButton = null;
```

Comment:
In my humble opinion, this naming seems to hard for predict about role.
Why don't you rename with `preparation` word?